pytorch_lightning/core/lightning.py

```diff
@@ -1211,6 +1211,25 @@ def on_save_checkpoint(self, checkpoint):
         """
         pass
 
+    def get_tqdm_dict(self):
```

Comment:
make it as property like a `tqdm_params`?

Reply:
Yeah, at first I made it property. But then I accidentally found out that it will cause misleading error messages, if there is an attribute error in the user code. What I mean, if you make 
```
@property
def tqdm_dict(self):
    _ = self.trainer.some_non_existing_attribute
    return {}
```
Then after 
```
return dict(**ref_model.tqdm_dict, **self.tqdm_metrics)
```
you will get
```
AttributeError: 'CoolSystem' object has no attribute 'tqdm_dict'
```
I don't know how to fix it, but it seems to be the known issue: https://stackoverflow.com/questions/15348331/why-does-property-decorator-show-object-has-no-attribute

Overall, I decided to stay without property, so the user gets the clear error message if there is some error in his code.